### PR TITLE
Adding Firefox 10 results

### PR DESCRIPTION
**Pull request by afmenez**
I tried Firefox 2, 3, 3.6 and 4, they all failed to display the results (probably due to #1127).  The earliest version I was able to run was Firefox 10 ESR.

### CODE REVIEW

File: data-es6.js [hunk 1]
```diff
@@ -7009,6 +7009,8 @@ exports.tests = [
         es6shim: true,
         ie11: true,
         firefox2: false,
+        firefox10: true,
```
Comment (zloirock) [inline data-es6.js]: Hmmm... Even in MDN I see that `Map` is available only from FF13... I'll check it.

Comment (zloirock) [reply data-es6.js]: image

**Review by zloirock — CHANGES_REQUESTED** [review]

File: data-es6.js (hunk 1 above)
Comment (afmenez) [reply data-es6.js]: Yep, and the the test fails on FF17.

Comment (zloirock) [reply data-es6.js]: It's FF10, result - `false`.

Comment (afmenez) [reply data-es6.js]: I have rerun the tests now and indeed it is false.  I'll fix the PR.

Comment (afmenez) [reply data-es6.js]: I see now what I did wrong, I mixed Map and WeakMap results.  I'll rerun everything and update the PR.

Comment (afmenez) [reply data-es6.js]: Fixed.

**Review by zloirock — APPROVED** [review]

**Pull request merged**